spec/stat_calculator_spec.rb

```diff
@@ -42,4 +42,27 @@
       expect(@stat_calculator.lowest_total_score).to eq(0)  
     end
   end
+
+  # Tests for win percentage calculations
+  describe '#percentage_home_wins' do
+    it 'calculates percentage_home_wins' do
+      # Check that the result is a Float (e.g., 66.67)
+      expect(@stat_calculator.percentage_home_wins).to be_a(Float)
```

Comment:
We want these tests to use the same values as our spec harness:
(the below tests and their values were taken directly from the spec harness. We should be using the same values shown here.)
ex) expect(stat_calculator.percentage_home_wins).to eq(0.44)

Spec harness tests:
  it "#percentage_home_wins" do
    expect(@stat_tracker.percentage_home_wins).to eq(0.44)
  end

  it "#percentage_visitor_wins" do
    expect(@stat_tracker.percentage_visitor_wins).to eq(0.36)
  end

  it "#percentage_ties" do
    expect(@stat_tracker.percentage_ties).to eq(0.20)
  end

Reply:
I currently have active pull requests for the methods assigned to me. If three-quarters of the team agrees to make further changes to the methods in the feature/stat_cal-win_percentage or feature/stat_cal_coach_performance branches, feel free to pull from origin, make the updates, and proceed as needed. This way, the team can continue moving forward without waiting on my availability.